openrtb_ext/imp_alkimi.go

```diff
@@ -0,0 +1,9 @@
+package openrtb_ext
+
+type ExtImpAlkimi struct {
+	Token      string  `json:"token"`
+	BidFloor   float64 `json:"bidFloor"`
+	Instl      int8    `json:"instl"`
+	Exp        int64   `json:"exp"`
+	AdUnitCode string  `json:"adUnitCode"`
```

Comment:
static/bidder-params/alkimi.json doesn't include description for `AdUnitCode`

Reply:
It's not required. AdUnitCode it's our internal parameter between PBS and Alkimi Exchange